### PR TITLE
7.0.x Add support for asynchronous tasks in mule2ballerina converter

### PR DESCRIPTION
**Pull request by arukshani**

### CODE REVIEW

File: tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/builder/BallerinaASTModelBuilder.java [hunk 1]
```diff
@@ -282,4 +283,28 @@ public void addComment(String comment) {
     public void addFunctionInvocationStatement(boolean argsAvailable) {
         modelBuilder.createFunctionInvocationStmt(null, null, nameReferenceStack.pop(), argsAvailable);
     }
+
+    public void createWorkerInvocationStmt(String workerName) {
+        modelBuilder.createWorkerInvocationStmt(workerName, null, null);
+    }
+
+    public void enterWorkerDeclaration() {
+        //  isWorkerStarted = true;
```
Comment (milindaperera) [inline tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/builder/BallerinaASTModelBuilder.java]: let's remove commented code

Comment (arukshani) [reply tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/builder/BallerinaASTModelBuilder.java]: This was fixed in this commit >> https://github.com/wso2/product-ei/pull/979/files#diff-e926d4a6b8bdff790cea82e4ff1ce334L329

File: tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/generator/Constants.java [hunk 2]
```diff
@@ -38,6 +38,7 @@
     public static final String FUNCTION_STR = "function";
     public static final String RETURN_STR = "return";
     public static final String STRUCT_STR = "struct";
+    public static final String WORKER_STR = "worker";
```
Comment (lafernando) [inline tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/generator/Constants.java]: Remove random whitespace.

File: tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/model/Flow.java [hunk 3]
```diff
@@ -67,10 +66,10 @@ public void accept(Visitor visitor) {
     @Override
     public void buildTree(DataCarrierDTO dataCarrierDTO) {
```
Comment (lafernando) [inline tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/model/Flow.java]: Remove whitespace, have to be consistent with the coding style.

**Pull request merged**